profiles/models.py

```diff
@@ -142,6 +148,12 @@ class Profile(models.Model):
     date_joined_micromasters = models.DateTimeField(blank=True, null=True, auto_now_add=True)
     linkedin = JSONField(blank=True, null=True)
     student_id = models.IntegerField(blank=True, null=True, unique=True)
+    currently_employed = models.CharField(
```

Comment:
We should store this as a boolean and update our frontend to send this value as a boolean also (#259). Can you revert this and we can do it in a separate PR?